controllers/sriovoperatorconfig_controller.go

```diff
@@ -99,6 +100,25 @@ func (r *SriovOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.
 		return reconcile.Result{}, err
 	}
 
+	// Check if we only have one node
```

Comment:
The same comment a for main.go change. Do we want to move this check to pkg.utils module as 'GetNodesConut' function?

Reply:
done can you please have another look?

Reply:
nit: i think reading L104 is pretty clear, you can drop this comment if you like.

Reply:
done